2021/fall/Sterne-TTh@10/oyunbileg-davaanyam.md

```diff
@@ -0,0 +1 @@
+# Hi I'm Oyu!
```

Comment:
This file seems good to me. I'm not sure about the second file though, it seems you pushed an empty file under the fall directory.

Reply:
Thanks for the comment! Deleted the empty file :)